server/backends/chunkstore/chunkstore.go

```diff
@@ -376,10 +392,12 @@ func (l *writeLoop) write() error {
 	if len(l.chunk) < l.chunkstore.writeBlockSize {
 		size = len(l.chunk)
 	}
+	fmt.Printf("Chunk size: %d, size: %d\n", len(l.chunk), size)
 	bytesWritten, err := l.chunkstore.writeChunk(l.ctx, l.blobName, l.chunkIndex, l.chunk[:size])
-	l.bytesFlushed += bytesWritten
-	l.chunk = l.chunk[bytesWritten:]
 	if bytesWritten > 0 {
+		fmt.Printf("bytesWritten: %d\n", bytesWritten)
```

Comment:
nit: log.Debugf instead of fmt.Printf (or just remove this)

Reply:
I'll just remove it, it's vestigial